gettsim/interface.py

```diff
@@ -152,6 +168,9 @@ def check_data_check_functions_and_merge_functions(
         needed by the user.
     data : dict of pandas.Series
         Data provided by the user.
+    aggregation_specs : dict, default None
+        A dictionary which contains specs for functions which aggregate variables on
```

Comment:
Again, refer to some place where I find useful information how this should look like. If I am lost before, I am also lost after reading this.